packages/experiments/src/components/Button/MenuButton/MenuButton.test.tsx

```diff
@@ -49,4 +50,26 @@ describe('MenuButton', () => {
     const tree = component.toJSON();
     expect(tree).toMatchSnapshot;
   });
+
+  it('focuses correctly when focus is triggered via IMenuButton interface', () => {
+    const wrapper = mount(
+      <div>
+        <MenuButton content="Menu Button 1" />
+        <MenuButton content="Menu Button 2" />
+        <MenuButton content="Menu Button 3" />
+      </div>
+    );
+
+    const buttons = wrapper.getDOMNode().querySelectorAll('button.ms-MenuButton') as NodeListOf<HTMLButtonElement>;
+    expect(buttons.length).toEqual(3);
+
+    buttons[0].focus();
```

Comment:
Same here. What is the `IMenuButton` interface if you're not getting it from `ref` or `componentRef`?